app/_hub/yesinteractive/kongmap/index.md

```diff
@@ -0,0 +1,138 @@
+---
+name: KongMap
+publisher: Yes Interactive
+
+categories:
+  - deployment
+
+type: integration
+
+desc: Visually Map and Manage your Kong Clusters
+
+description: |
+  Kongmap is a free visualization tool which allows you to view and edit configurations of your Kong API Gateway Clusters, including Routes, Services, and Plugins/Policies. The tool is being offered for installation via Docker and Kubernetes at this time. 
```

Comment:
https://www.grammarly.com/blog/which-vs-that/#:~:text=In%20a%20defining%20clause%2C%20use,and%20you%20can%20use%20which.

```suggestion
  Kongmap is a free visualization tool that allows you to view and edit configurations of your Kong API Gateway Clusters, including Routes, Services, and Plugins/Policies. The tool is being offered for installation using Docker and Kubernetes at this time. 
```